src/devices/Hmc5883l/Hmc5883l.cs

```diff
@@ -120,5 +138,33 @@ public void Dispose()
                 _sensor = null;
             }
         }
+
+        /// <summary>
+        /// Reads device statuses .
+        /// </summary>
+        /// <returns>IList<Status></returns>
+        private IList<Status> GetStatus()
+        {
+            var result = new List<Status>();
+            _sensor.WriteByte((byte)Register.HMC_STATUS_REG_ADDR);
+            byte status = _sensor.ReadByte();
+
+            byte mask = 0b00000001;
```

Comment:
NIT instead of having a variable, can you instead just do the inline check like: `if ((status & 0b_0001) != 0) {` etc.

Reply:
Fixed